content/docs/concepts/jobs_instances.md

```diff
@@ -43,4 +43,13 @@ the following time series:
 * `scrape_series_added{job="<job-name>", instance="<instance-id>"}`:
    the approximate number of new series in this scrape. *New in v2.10*
 
+if you enable feature [extra-scrape-metrics](/docs/prometheus/latest/feature_flags/#extra-scrape-metrics), you will also get:
+
+* `scrape_timeout_seconds{job="<job-name>", instance="<instance-id>"}`:
+  the configured scrape_timeout for a target.
```

Comment:
Either "`scrape_timeout`" as code or just "scrape timeout".

Same for the sample_limit below.